koa-crud/src/graphql/typeDefs/promo-enrollments.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from 'apollo-server-koa';
+
+export default gql`
+  extend type Mutation {
+    enrollToPromo(input: EnrollToPromoInput!): Boolean
+  }
+
+  input EnrollToPromoInput {
+    promo: String!
+  }
+`;
```

Comment:
If there is only one argument at best then you can directly put it. No need to add an `input`. You can do it like this:
```gql
extend type Mutation {
    enrollToPromo(promo: ID!): Boolean!
}
```

Reply:
Noted sir